consent/strategy_default.go

```diff
@@ -697,7 +697,14 @@ func (s *DefaultStrategy) executeBackChannelLogout(ctx context.Context, r *http.
 	}
 
 	var wg sync.WaitGroup
-	hc := httpx.NewResilientClient()
+	hc := httpx.NewResilientClient(
+		httpx.ResilientClientWithClient(
+			&http.Client{
+				Timeout: time.Minute,
+				CheckRedirect: func(req *http.Request, via []*http.Request) error {
+					return http.ErrUseLastResponse // This results in no retry. If retry is favorable, then custom error should be thrown. If redirects should be allowed, then CheckRedirect should be made configurable (other configuration related PRs #2875, #2849).
```

Comment:
> Not sure if I agree on disabling redirects. Sometimes we use redirects when API endpoints move in Ory so it's a common use case in our own system.
Ok that makes sense!
 
> Also I'm not following 100%, this only disables following redirects, it doesn't disable retries right?

Considering redirects have actual use case I could make it configurable:

a) Redirect not expected. Request should **not** be retried -> return http.ErrUseLastResponse;
b) Logout endpoint is not expected to make redirect, but for some reason it redirects. Request **should** be retried -> return errors.New("Unexpected redirect")
c) Logout endpoint is allowed to make redirect -> CheckRedirect is not configured

All other errors are retried in all cases!

Reply:
Thank you for the reply and sorry for my late response. I'm not sure if we want this configurable, it's quite the edge case and it's not very common to retry 3xx responses. It's much more common to retry 500, 502, etc. Some respone codes like 400 even explicitly express: "do not retry because the request is malformed".

I would like to keep the current behavior because I think it is correct behavior. If something is misconfigured, would retrying the 3xx error within short succession (e.g. 500ms) really help? I would expect that we just end up in another 3xx redirect if we retry, and very little will be gained. Do you agree?